project/src/components/private-route/private-route.tsx

```diff
@@ -0,0 +1,20 @@
+import {Navigate} from 'react-router-dom';
+import {RouteProps} from 'react-router-dom';
+import {AppRoute, AuthorizationStatus} from '../../const';
+
+type PrivateRouteProps = RouteProps & {
+  authorizationStatus: AuthorizationStatus;
+  children: JSX.Element;
+}
+
+function PrivateRoute(props: PrivateRouteProps): JSX.Element {
```

Comment:
Сразу доставай значения из пропсов
```suggestion
function PrivateRoute({authorizationStatus, children}: PrivateRouteProps): JSX.Element {
```